site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -452,6 +452,29 @@ public function changeRegex($queue_code) {
         );
     }
 
+    /**
+     * @Route("/courses/{_semester}/{_course}/office_hours_queue/{queue_code}/change_contact_information", methods={"POST"})
+     * @AccessControl(role="LIMITED_ACCESS_GRADER")
+     * @return MultiResponse
+     */
+    public function changeContactInformation($queue_code) {
+        if (empty($queue_code)) {
```

Comment:
Should this be `empty` or `isset` 
https://stackoverflow.com/a/1219550

Reply:
yup should be !isset